tt_eager/tt_dnn/op_library/prod/kernels/utils.hpp

```diff
@@ -0,0 +1,298 @@
+// SPDX-FileCopyrightText: © 2023 Tenstorrent Inc.
+//
+// SPDX-License-Identifier: Apache-2.0
+
+#include <stdint.h>
+
+#include "dataflow_api.h"
+
+void fill_cb_with_value(uint32_t cb_id, uint32_t value, int32_t num_of_elems = 1024) {
+    cb_reserve_back(cb_id, 1);
+    auto ptr = reinterpret_cast<uint16_t *>(get_write_ptr(cb_id));
+    for (int j = 0; j < num_of_elems; j++) {
+        ptr[j] = uint16_t(value >> 16);
+    }
+    cb_push_back(cb_id, 1);
+}
```

Comment:
Any reason why we use int for num_of_elems/loop?

For pointers to l1 memory you should create them as
`volatile tt_l1_ptr std::uint16_t* ptr = (volatile tt_l1_ptr uint16_t*)(get_write_ptr(cb_id));`
This is to avoid a potential hang due to HW bug.

Can also store/use the u16 value like how you do it in later code
`const auto u16_value = uint16_t(value >> 16);`

Reply:
Functions in this file are not called anywhere inside PROD. So I'll delete this file.